js/package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "prettier --single-quote --trailing-comma es5 --print-width 150 --tab-width 4 --write src"
   },
   "devDependencies": {
-    "flarum": "0.1.0-beta.16",
+    "flarum": "^1.0.0",
```

Comment:
We're deprecating this package, so it can be removed.

Don't forget to `npm i` in the JS folder and commit the new lockfile :)

```suggestion
```